metabase/metabase
# Change search input placeholder depending on selected folder & tab in Entity Picker

Closes #47290
Closes #47292

### Description

Tests will be added later.

### How to verify

1. New > Question
2. Select a collection or a database or a schema 

Search input placeholder should update accordingly.

If database has only 1 schema, the placeholder should mention the database, not the schema.

### Demo

https://github.com/user-attachments/assets/74cb8d67-b758-4691-ad3b-a1445aeef05e
